wiki/quest-modding-bmbf.md

```diff
@@ -0,0 +1,204 @@
+---
+prev: false
+next: false
+---
+
+# Quest Modding using BMBF
+
+## Before You Begin
+
+If you haven't already, go to [Preparations](./quest-modding.md#preparations) and follow those steps.
+You should now have Developer Mode enabled and SideQuest Advanced installed on your PC.
+It's recommended to tape up the proximity sensor between the lenses of your Quest. This prevents it from
```

Comment:
FYI there is a button in sidequest to turn off the proximity sensor which should probably be suggested instead